ansible_ai_connect/ai/api/model_pipelines/tests/test_factory.py

```diff
@@ -91,6 +99,20 @@ def test_default_fallback_playbook_explanation(self):
                 log,
             )
 
+    # 'dummy' does not have a ChatBot pipeline
+    @override_settings(ANSIBLE_AI_MODEL_MESH_CONFIG=json.dumps(CHATBOT))
```

Comment:
We can do without the `json.dumps` call here.

Reply:
`ANSIBLE_AI_MODEL_MESH_CONFIG` must be a `string` containing either JSON or YAML.

Using a `dict` is not supported (remember, the value is loaded from an environment variable).

Reply:
I mean, `CHATBOT` can just be a plain YAML string :-)

Reply:
I mean, `CHATBOT` can just be a plain YAML string :-)